### ISSUE TITLE
schema  linker 模型微调不能复现结果

### ISSUE THREAD

**Issue by dshwei** [RUCKBReasoning/codes#22]
使用 schema linker 微调的结果，和您开源的sql  generate 模型，进行测试：

schema  linker 微调的脚本如下：
   deepspeed  --master_port 29567 train_schema_item_filter.py \
    --batch_size 4 \
    --gradient_descent_step 8 \
    --device 0 \
    --learning_rate 1e-5 \
    --gamma 2.0 \
    --alpha 0.75 \
    --epochs 64 \
    --patience 8 \
    --seed 42 \
    --save_path ./sic_ckpts/sic_bird_with_evidence \
    --tensorboard_save_path ./train_logs/sic_bird_with_evidence \
    --train_filepath ./sft_bird_with_evidence_train_text2sql.json \
    --dev_filepath ./sft_bird_with_evidence_dev_text2sql.json \
    --model_name_or_path roberta-large \
    --mode train

sic_bird_with_evidence   +   seeklhy/codes-7b-bird

发现测试结果没有达到 57.63,只有  50.46
  

![企业微信截图_17183526669380](https://github.com/RUCKBReasoning/codes/assets/42167236/5fb41c17-0c51-45ff-bf3e-2aba85f11a8f)

通过对比发现，schema linker 模型微调不能复现论文里的效果。

**Comment by lihaoyang-ruc** [comment]
schema linker 在dev集上的auc是多少？
